KDCoin/spvClient.py

```diff
@@ -1,3 +1,32 @@
+<<<<<<< HEAD
+from KDCoin.transaction import Transaction
+from KDCoin.keyPair import GenerateKeyPair
+
+# Single Client
+# Functionalities: i) Create Transactions, ii) Verify transactions validated
+
+receiver_private_key, receiver_public_key = GenerateKeyPair()
+print(type(receiver_public_key))
+
+class SPVClient:
+
+    def __init__(self):
```

Comment:
please init this way instead:
def __init__(self, _pub, _priv)

GenerateKeyPair should only be called as a first time user trying to generate from the App